cluster/gce/util.sh

```diff
@@ -833,6 +844,9 @@ function create-subnetworks() {
         else
           echo "${color_yellow}Using pre-existing network ${NETWORK}, subnets won't be expanded to /19!${color_norm}"
         fi
+      elif [[ "${CREATE_CUSTOM_NETWORK:-}" == "true" ]]; then
+        assert-node-ip-range
+        gcloud compute networks subnets create "${NETWORK}" --project "${NETWORK_PROJECT}" --region "${REGION}" --network "${NETWORK}" --range "${NODE_IP_RANGE}"
```

Comment:
We should question whether we want to name the subnet after the network. May help people debug issues if they're different and make it apparent that the network is type manual.

Reply:
Good point, I named the subnet this way in order to utilize the existing cleanup logic in `delete-subnetworks()`. Will take a look if I could make it neat.

Reply:
Changed to `${NETWORK}-subnet-custom`.